### PR TITLE
ACC-978 Add consumer group config

### PR DESCRIPTION
**Pull request by abramin**

### CODE REVIEW

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: dev-aws/kafka-shared-msk/account-identity
Commit ID: 9cf2f92a0e8633daccb09f886f951d5c30452f40
Requested At: 2024-10-01T14:55:41Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/account-identity` as comment.

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: dev-aws/kafka-shared-msk/account-identity
Commit ID: 9cf2f92a0e8633daccb09f886f951d5c30452f40
Requested At: 2024-10-01T14:55:41Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/account-identity` as comment.

File: dev-aws/kafka-shared-msk/account-identity/account.tf [hunk 1]
```diff
@@ -143,6 +143,13 @@ module "account_identity_account_api_atomic_projector" {
   cert_common_name = "account-platform/account_api_atomic_projector"
 }
 
+module "account_identity_account_api_change_notifier" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = [kafka_topic.account_identity_account_atomic_v1.name]
+  consume_groups   = ["account-identity.account-api-change-notifier"]
+  cert_common_name = "account-platform/account_api_atomic_projector"
```
Comment (Nesze) [inline dev-aws/kafka-shared-msk/account-identity/account.tf:150]: should this be renamed?

Comment (abramin) [reply dev-aws/kafka-shared-msk/account-identity/account.tf:150]: The idea was to reuse a certificate if possible so we don´t need completely separate certificates for every single deployment.
In this case both these services are part of the same system and consume the same topic so thought it made sense.

Comment (Nesze) [reply dev-aws/kafka-shared-msk/account-identity/account.tf:150]: Makes sense. Just noticed we did it differently in the verification system - and thought will follow suit.

But I agree, we can keep these the same. Let's keep an 👁️ out for consistency.

Comment (abramin) [reply dev-aws/kafka-shared-msk/account-identity/account.tf:150]: The cert name should be more generic like "account-api" so it makes to reuse across all account-api subsystems

Comment (Nesze) [reply dev-aws/kafka-shared-msk/account-identity/account.tf:150]: Let's fix it then.

**Pull request merged**